debug_module.tex

```diff
@@ -100,10 +100,10 @@ \section{Selecting Harts} \label{selectingharts}
 \subsection {Selecting Multiple Harts} \label{hartarraymask}
 
 Debug Modules may implement a Hart Array Mask register to allow selecting
-multiple harts at once. The Nth bit in the Hart Array Mask register is used to
-select the hart with index N. Usually a DM will have a Hart Array Mask register
-exactly wide enough to select all the harts it supports, but it's allowed to
-tie any of these bits to 0.
+multiple harts at once. The Nth bit in the Hart Array Mask register aplies to
```

Comment:
applies

Reply:
Thanks. Evidently I need to look into a spell checker.